app/models/transfer.rb

```diff
@@ -56,6 +56,11 @@ def csv_export_attributes
     ]
   end
 
+  def deletable?
+    item_ids = line_items.pluck(:item_id)
+    Audit.where(storage_location_id: [to_id, from_id]).where(created_at: created_at..).joins(:line_items).where(line_items: {item_id: item_ids}).empty?
```

Comment:
Can we split this line up to make it more readable?

Reply:
Also, I'm not sure this belongs inside `Transfer` since it really isn't checking transfers at all. I think it'd make more sense to move this to the `Audit` class as a method, e.g. `Audit.since?(self)` and it can take an `Itemizable` as its parameter.